bioswagger/bioswagger.py

```diff
@@ -0,0 +1,104 @@
+from content.dna_rna_tools import reverse, reverse_complement, transcribe, complement, is_valid_seq
+from content.fastq_functions import count_gc, get_seq_quality
+import os
+
+def read_fastq(fastq_file) -> dict:
+    """
+    read file and transforms it into dict
+    :return:
+    """
+    # no fastq file check
+    with open(fastq_file) as fastq:
+        parsed_fq = {}
+        # deleting exceed symbols
+        seqs = [line.strip() for line in fastq.readlines()]
+        if seqs % 4 != 0:
+            raise ValueError("Broken FASTQ! Lines dont divided by 4!")
+        # lets divide by 4 our lines via zip
+        # make iterator based on our list
+        it = iter(seqs)
+
+        for name, sequence, info, quality in zip(it, it, it, it):
+            # write to dict
+            parsed_fq[name] = (sequence, quality)
+        # в блоке или нет?
+        return parsed_fq
+
+
+
+def write_fastq(seqs: dict, output_suffix: str):
+    """
+    Write file into a folder 'filtered' with prename suffix 'output_fastq' given
+
+    """
+    # checking folder
+    if not os.path.isdir('filtered'):
+        os.mkdir('filtered')
+    #to fitered folder
+    
+    pass
+
+
+def run_dna_rna_tools(*input_list) -> list:
+    """
+    Takes list of seqs with method as last argument
+    :param input_list: list
+    :return: list
+    """
+    # переписать тут, чтобы было как на консе у антона
+    *input_list, method = input_list

```

Comment:
Можно было бы сразу задать *input_list, method, когда создавал функцию: 
```suggestion
    def run_dna_rna_tools(*input_list -> list, method -> str) ->list:

```